### PR TITLE
check user is admin before search criteria filtration

### PR DESCRIPTION
**Pull request by anivirtoway**

### CODE REVIEW

File: VirtoCommerce.OrderModule.Web/Security/OrderReadPricesPermission.cs [hunk 1]
```diff
@@ -9,21 +9,17 @@ public static class OrderReadPricesPermission
...
             {
                 const CustomerOrderResponseGroup val = CustomerOrderResponseGroup.Full & ~CustomerOrderResponseGroup.WithPrices;
-
                 result = val.ToString();
             }
             else if (needRestrict)
             {
                 var items = respGroup.Split(',').Select(x => x.Trim()).ToList();
-
                 items.Remove(CustomerOrderResponseGroup.WithPrices.ToString());
```
Comment (yecli) [inline VirtoCommerce.OrderModule.Web/Security/OrderReadPricesPermission.cs]: do not need to remove line between declaration and statement, if I remember correctly

File: VirtoCommerce.OrderModule.Web/Security/OrderReadPricesPermission.cs [hunk 2]
```diff
@@ -9,21 +9,17 @@ public static class OrderReadPricesPermission
...
         {
             var result = respGroup;
-
             var needRestrict = permissions.Any() && !permissions.Any(x => x.Id == OrderPredefinedPermissions.ReadPrices);
 
             if (needRestrict && string.IsNullOrWhiteSpace(respGroup))
             {
                 const CustomerOrderResponseGroup val = CustomerOrderResponseGroup.Full & ~CustomerOrderResponseGroup.WithPrices;
-
                 result = val.ToString();
```
Comment (yecli) [inline VirtoCommerce.OrderModule.Web/Security/OrderReadPricesPermission.cs]: do not need to remove line between declaration and statement, if I remember correctly

Comment (anivirtoway) [reply VirtoCommerce.OrderModule.Web/Security/OrderReadPricesPermission.cs]: reverted

File: VirtoCommerce.OrderModule.Web/Controllers/Api/OrderModuleController.cs [hunk 3]
```diff
@@ -560,31 +560,37 @@ public IHttpActionResult GetOrderChanges(string id)
         private CustomerOrderSearchCriteria FilterOrderSearchCriteria(string userName,
             CustomerOrderSearchCriteria criteria)
         {
-            if (!_securityService.UserHasAnyPermission(userName, null, OrderPredefinedPermissions.Read))
+            var user = _securityService.FindByNameAsync(userName, UserDetails.Reduced).Result;
+            if (!user.IsAdministrator)
```
Comment (tatarincev) [inline VirtoCommerce.OrderModule.Web/Controllers/Api/OrderModuleController.cs]: By design `!_securityService.UserHasAnyPermission ` must always returns true for account with set IsAdministrator flag, please rework your changes in accordance with this information

Comment (anivirtoway) [reply VirtoCommerce.OrderModule.Web/Controllers/Api/OrderModuleController.cs]: done

Comment (yecli) [reply VirtoCommerce.OrderModule.Web/Controllers/Api/OrderModuleController.cs]: Checked, it works as expected.
@tatarincev Please review.

**Pull request merged**